app/components/CoinListCell/CoinListCellChangeButton/CoinListCellChangeButton.js

```diff
@@ -0,0 +1,58 @@
+import React, { Component } from 'react'
+import { View, Text, TouchableOpacity } from 'react-native'
+import PropTypes from 'prop-types'
+import coinListCellChangeButtonStyles from './CoinListCellChangeButtonStyles'
+
+export default class CoinListCellChangeButton extends Component {
```

Comment:
I know in `CoinListCellChangeButton`, the change part refers to a noun but it reads as an action, meaning it's a button to change `coin list cell`. What do you think about calling it `CoinListCellDeltaButton`?
Just a suggestion. I'm not sure if I like the inconsistency with the API but once again, if we're remapping the data, we can name that field `delta` ourselves for more clarity.

Reply:
I like that.